### ISSUE TITLE
Lasso Tool

### ISSUE THREAD

**Issue by Aborres** [serge-rgb/milton#28]
Hi! Just sharing an idea. It would be great to have a lasso tool like photoshop has to selected certain areas and be able to move them or copy/paste them in other places in the canvas.

Thank you!

**Comment by serge-rgb** [comment]
I agree that a lasso tool or any kind of way to select/cut/copy/paste would be very useful. I haven't given much thought to how this could be implemented in Milton but it's worth looking into

**Comment by Aborres** [comment]
I've been using milton for a few weeks and is fantastic, well done. I was thinking about user experience and maybe could be useful to have keybinding. In most graphics programs the color picker is in the alt key when you are using the brush for example.

**Comment by serge-rgb** [comment]
Glad to hear that you like Milton and that it has proven useful :D 
I was supposed to implement Alt-key-toggles-eyedropper a while back but I totally forgot.. Currently the 'i' key can be used, but it doesn't toggle. 
I will add the Alt key for next release. The lasso tool or whatever selection method I end up implementing probably will take more than one release.

**Comment by jitspoe** [comment]
+1 for a lasso+move tool.  That would greatly help my workflow.  I feel like it's currently the biggest feature missing right now.
